### PR TITLE
Document device type configurable

### PR DESCRIPTION
**Pull request by lluiscampos**
Document [PR 393](https://github.com/mendersoftware/mender/pull/393) from mender client and minor beautification.

### CODE REVIEW

File: 05.Client-configuration/05.Configuration-file/50.Configuration-options/docs.md [hunk 1]
```diff
@@ -106,6 +106,11 @@ if using Hosted Mender.
 The location where deployment logs will be written. This must be on a persistent
 partition to work correctly.
 
+#### DeviceTypeFile
+
+The location where to to store de device_type. This must be on a persistent
+partition to work properly. Default location is `/var/lib/mender/device_type`
```
Comment (mirzak) [inline 05.Client-configuration/05.Configuration-file/50.Configuration-options/docs.md]: I would reword this slightly, "work properly" does not really state anything :).

Something like,

"Our recommendation is to store the `device_type` in a persistent location to avoid it accidentally being changed due to an root filesystem update."

Comment (lluiscampos) [reply 05.Client-configuration/05.Configuration-file/50.Configuration-options/docs.md]: Thanks. I wanted to be consistent with the previous item, so I will change both ;)

**Comment by lluiscampos** [comment]
@mirzak Reworked.

I kept the "must" because I think it should be stronger than just a recommendation. I rewrote the "fix properly" part, though.

File: 05.Client-configuration/05.Configuration-file/50.Configuration-options/docs.md [hunk 2]
```diff
@@ -104,7 +104,13 @@ if using Hosted Mender.
 #### UpdateLogPath
 
 The location where deployment logs will be written. This must be on a persistent
-partition to work correctly.
+partition to to avoid losing the logs due to an root filesystem update.
+
+#### DeviceTypeFile
+
+The location where to to store de device_type. This must be on a persistent
```
Comment (mirzak) [inline 05.Client-configuration/05.Configuration-file/50.Configuration-options/docs.md]: `to to` -> `to`

and

`de` -> `the`

File: 05.Client-configuration/05.Configuration-file/50.Configuration-options/docs.md [hunk 3]
```diff
@@ -104,7 +104,13 @@ if using Hosted Mender.
 #### UpdateLogPath
 
 The location where deployment logs will be written. This must be on a persistent
-partition to work correctly.
+partition to to avoid losing the logs due to an root filesystem update.
```
Comment (mirzak) [inline 05.Client-configuration/05.Configuration-file/50.Configuration-options/docs.md]: `to to` -> `to`

File: 05.Client-configuration/05.Configuration-file/50.Configuration-options/docs.md [hunk 4]
```diff
@@ -104,7 +104,13 @@ if using Hosted Mender.
...
+
+#### DeviceTypeFile
+
+The location where to to store de device_type. This must be on a persistent
+partition to avoid accidentally being changed due to an root filesystem update.
```
Comment (mirzak) [inline 05.Client-configuration/05.Configuration-file/50.Configuration-options/docs.md]: `to avoid accidentally` -> `to avoid it accidentally`

**Comment by lluiscampos** [comment]
Embarrassing typos :weary: . Thanks.

Reworked.
